run_length_encoding/run_length.erl

```diff
@@ -1,4 +1,25 @@
 -module(run_length).
--compile(exports_all).
 
-	      
+-export([run_length_encode/1]).
+
+-include_lib("eunit/include/eunit.hrl").
```

Comment:
You may consider writing test in a separate folder `test`.

Reply:
+1 also let's stop using eunit and do all the tests with common test